torch/csrc/jit/passes/quantization/helper.cpp

```diff
@@ -295,9 +295,10 @@ std::vector<Value*> getPassThroughInputs(Value* v) {
       inputs.push_back(output);
     }
     return inputs;
-  } else if (n->kind() == prim::ListUnpack) {
+  } else if (n->kind() == prim::ListUnpack || n->kind() == prim::TupleUnpack) {
```

Comment:
Can you explain why you return input(0) here?

Reply:
input(0) is the input of the list unpack/ tuple unpack. which is a list/tuple. this function is to propagate observed property from input to output, in this case we can propagate the observed property from the list/tuple to their output, which are the unpacked values.